js/main.js

```diff
@@ -1,20 +1,57 @@
-function getRandomIntInclusive (min, max) {
-  let result;
-  if (min < 0 || max < 0 || max <= min) {
-    result = 'Проверьте, пожалуйста, введенный диапазон чисел: диапазон может быть только положительный, включая ноль, а максимальное значение должно превышать минимальное.';
-  } else {
-    min = Math.ceil(min);
-    max = Math.floor(max);
-    result = Math.floor(Math.random() * (max - min + 1)) + min;
-  }
-  return result;
+const COMMENT_DESCRIPTIONS = ['Пляж отеля', 'Go to the beach', 'Цвет моря', 'Подруга', 'Супчик', 'Макларен', 'Десерт', 'Вкусный напиток', 'Летают самолеты', 'Храним обувь', 'Путь к морю', 'Ауди', 'Ужин', 'Пункт меню', 'Отдыхаю', 'Лечу в отпуск', 'Концерт', 'Ретро-машина', 'Тапочки с подсветкой', 'Пальмы', 'Вилка', 'Закат', 'Краб', 'Руки на концерте', 'Бегемот',];
+const COMMENT_NAMES = ['Дарья', 'Алексей', 'Иван', 'Наталья', 'Юлия', 'Андрей',];
+const MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.', 'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.', 'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.', 'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.', 'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',];
+const SIMILAR_DESCRIPTIONS_COUNT = 25;
+const MAX_COMMENT_COUNT = 4;
+const idMessageArray = [];
```

Comment:
```suggestion
const MAX_COMMENT_COUNT = 4;

const idMessageArray = [];
```

логические блоки лучше разделять пустой строкой.

Reply:
Логический блок - это когда объявление констант и массивов, относящихся к какой-то функции, отделяется визуально пустой строкой от объявления констант и массивов, относящихся к другой функции? Т.е. в данном случае мне достаточно отделить объявление пустых массивов от объявления констант выше или необходимо их перетасовать друг с другом и сгруппировать по блокам по совместному использованию в какой-то функции? А то информации много новой и я пока плаваю в простых вещах.